demo/addons/godot-openxr/scenes/first_person_controller_vr.gd

```diff
@@ -35,8 +48,12 @@ func initialise() -> bool:
 		# Find the viewport we're using to render our XR output
 		var vp : Viewport = _get_xr_viewport()
 
+		# Obtain enabled extensions
+		enabled_extensions = $Configuration.get_enabled_extensions()
+
 		# Start passthrough?
-		_start_passthrough()
+		if enable_passthrough:
+			_start_passthrough()
```

Comment:
Here you should follow the same pattern as in `set_passthrough(...)` and update `enable_passthrough` with the result from `_start_passthrough`.

Reply:
Ok, thats a misunderstanding.

These are properties so the values are stored in the TSCN file. After loading Godot will call set_passthrough with the stored value however this is called as part of the loading process when we're not yet ready to call `_start_passthrough`. That is why we call `_start_passthrough` here based on whether the setting was set during load.

While I used to do a syntax here like `set_passthrough(enable_passthrough)`, I've stopped doing this because in Godot 4 the syntax for properties changed and this would result in a never ending loop as setting `enable_passthrough` would call `set_passthrough` again.